torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -197,6 +197,32 @@ def my_slow_method(self, my_tensor_arg):
         time.sleep(5)
         return torch.add(self.a, my_tensor_arg)
 
+class Server:
+    @staticmethod
+    @rpc.functions.async_execution
+    def identity(tensor):
+        fut = torch.futures.Future()
+        fut.set_result(tensor)
+        return fut
```

Comment:
Why is this using async execution? Couldn't it just return the argument?

Reply:
Yes for testing it could just return the argument. This code was removed.